### ISSUE TITLE
LineEdit makes fps drop when moving cursor on it if you have a ViewportContainer with a Viewport in the same scene.

### ISSUE THREAD

**Issue by AilerNX** [godotengine/godot#50675]
### Godot version

3.3.2.stable.official

### System information

Windows 10, GLES 3, Nvidia MX110

### Issue description

When you have a ViewportContainer with a Viewport inside and a LineEdit on same scene just like this:

![scenetree](https://user-images.githubusercontent.com/28108603/126410057-b0e7fe56-020e-46a7-acda-4036401cda91.png)

The FPS drops if you keep moving the cursor inside of LineEdit.

### Steps to reproduce

Just keep moving the cursor inside LineEdit rect having ViewportContainer with a Viewport inside.

### Minimal reproduction project

I made a scene with a Label printing the delta on screen to see it better.

[LineEdit.zip](https://github.com/godotengine/godot/files/6851842/LineEdit.zip)

**Comment by Calinou** [comment]
See also https://github.com/godotengine/godot/issues/39608. This is likely because the mouse cursor keeps changing back and forth.

**Comment by AilerNX** [comment]
Yeah the cursor keeps flickering and changing the shape between Arrow and Ibeam, but this doesn't happen without the Viewports behind.

**Comment by AilerNX** [comment]
I noticed now that is any Control node setted with the Ibeam or Cross cursor shapes make this happen.

**Comment by Calinou** [comment]
@MennoMax Can you try to [run a profiler on a debug build](https://docs.godotengine.org/en/latest/development/cpp/using_cpp_profilers.html) when this happens? This way, we can see what's taking the most CPU time.

**Comment by MennoMax** [comment]
I'm very sorry, had the wrong tab open. Meant to report this to #48309 ^^

There's no fps drop for me on Linux.

**Comment by AilerNX** [comment]
> Meant to report this to #48309

Now I think this issue here is related or even a duplicate of #48309

> There's no fps drop for me on Linux

I haven't tested on Linux but on Windows the FPS drops when I keep moving the cursor inside a LineEdit or any Control Node with the Ibeam or Cross shapes setted

>  This is likely because the mouse cursor keeps changing back and forth every frame

But like Calinou said this is probably the cause of my FPS drop, my workaround was to change the cursor shape from the LineEdit to another instead of the Ibeam or Cross, which are the only ones that make the cursor flicker.
